docker/prod/build.sh

```diff
@@ -1,9 +1,59 @@
 #!/bin/bash
+# Before run of this script you can set environmental variables
+# IMAGE_TAG ... then  export them
+# and to use defined values instead of default ones
 
 cd "$(dirname "$0")"
 
 set -e
 
-IMAGE_TAG=${IMAGE_TAG:-'prod_vpp_agent'}
+#To prepare for future fat manifest image by multi-arch manifest,
+#now build the docker image with its arch
+#For fat manifest, please refer
+#https://docs.docker.com/registry/spec/manifest-v2-2/#example-manifest-list
 
-sudo docker build  ${DOCKER_BUILD_ARGS} --tag ${IMAGE_TAG} .
+BUILDARCH=`uname -m`
+
+case "$BUILDARCH" in
+  "aarch64" )
+    IMAGE_TAG=${IMAGE_TAG:-'prod_vpp_agent-arm64'}
+    # Dockerfile  for prod_vpp_agent expects that dev-vpp-agent:latest image is built
+    docker tag dev_vpp_agent-arm64:latest dev_vpp_agent:latest
```

Comment:
Now I see you are retagging here, but this just unnecessarily complicates overall build process. Always try to minimize parts that don't really help, this just make script longer with extra commands to execute and actually does nothing useful. Keep local image name just `dev_vpp_agent` and `prod_vpp_agent`.

Reply:
ok